src/components/GuideWrapper.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import GuideBlock from './GuideBlock';
+
+function GuideWrapper(props) {
```

Comment:
```suggestion
function GuideWrapper({guides}) {
```
Clearer to have it showing the props passed in. Easier to glance at it in future and understand it.